yarn-project/end-to-end/src/e2e_token_contract.test.ts

```diff
@@ -641,6 +679,44 @@ describe('e2e_token_contract', () => {
           );
           expect(await asset.methods.balance_of_private(accounts[0].address).view()).toEqual(balance0);
         });
+
+        it('transfer on behalf of other, cancelled approval', async () => {
```

Comment:
```suggestion
        it('transfer on behalf of other, cancelled authwit', async () => {
```

Autistic nit but for the test in public you use "authwit" and not "approval". Would keep the naming consistent.